### PR TITLE
Set cloud provider type in AVS evaluation names correctly

### PR DESCRIPTION
**Pull request by ebensom**
**Description**

The AVS evaluation names have to reflect the correct cloud provider type correctly:

- for Azure it should be AZR
- for AWS it's AWS

Added shoot name to the AVS evaluation description.

### CODE REVIEW

File: components/kyma-environment-broker/internal/avs/model.go [hunk 1]
```diff
@@ -116,14 +116,35 @@ func newBasicEvaluationCreateRequest(operation internal.ProvisioningOperation, e
...
 
+func providerCodeByPlan(planID string) string {
+	switch planID {
+	case broker.AWSPlanID:
+		return "AWS"
+	case broker.GCPPlanID:
+		return "GCP"
+	case broker.AzurePlanID, broker.AzureLitePlanID, broker.AzureHAPlanID:
+		return "AZR"
+	case broker.TrialPlanID, broker.FreemiumPlanID:
```
Comment (piotrmiskiewicz) [inline components/kyma-environment-broker/internal/avs/model.go:139]: freemium could be AWS or Azure

**Comment by piotrmiskiewicz** [comment]
We are planning to have an explicit value in an instance or operation which defines the provider (calculated provider) and add it to an info in /runtimes endpoint.

**Review by piotrmiskiewicz — APPROVED** [review]

File: components/kyma-environment-broker/internal/avs/model.go (hunk 1 above)
Comment (PK85) [reply components/kyma-environment-broker/internal/avs/model.go:139]: exactly, I think we should do first https://github.tools.sap/kyma/backlog/issues/1473 and then adjust and merge this PR.

**Pull request merged**